test/mason/mason-external/libtomlc99/mason-external.good

```diff
@@ -1,4 +1,6 @@
+  Fetch: <time>.  Build: <time>.  Total: <time>.
 Installing Spack backend ...
+Spack installation successful
```

Comment:
Do we still print this message? When running this test locally, I got an error because this line was missing.

Reply:
I think you had mentioned in a previous comment that nothing else should be printed other than error messages during spack installation, so I removed that. Would you want that to be added back?

Reply:
Gotcha - I think it's fine to leave it out. Let's just update this `.good` :)

Reply:
Also, should I get rid of the `SKIPIF` from the test now ?

Reply:
Let's have me do that in a follow-up PR after this is merged. I will need to test manually on several of our systems before we do that to catch any unexpected issues.

Reply:
Cool !